packages/at_client/lib/src/decryption_service/self_key_decryption.dart

```diff
@@ -19,7 +20,16 @@ class SelfKeyDecryption implements AtKeyDecryption {
           intent: Intent.decryptData,
           exceptionScenario: ExceptionScenario.decryptionFailed);
     }
-
+    if (atKey.key == "shared_key") {
```

Comment:
The copy alice will create for themselves of a shared key they cut for bob will be `shared_key.bob@alice`. Will this test work? What happens when you use this branch with your REPL?